tests/integration/util/snapshot_test_utils.py

```diff
@@ -1,39 +0,0 @@
-import io
-
-from PIL import Image
```

Comment:
If we're removing this, lets also kill it in setup.py https://github.com/DataDog/datadogpy/blob/master/setup.py#L25